src/py/flwr/superexec/exec_servicer.py

```diff
@@ -60,11 +61,16 @@ def __init__(
         linkstate_factory: LinkStateFactory,
         ffs_factory: FfsFactory,
         executor: Executor,
+        auth_plugin: Optional[ExecAuthPlugin] = None,
     ) -> None:
         self.linkstate_factory = linkstate_factory
         self.ffs_factory = ffs_factory
         self.executor = executor
         self.executor.initialize(linkstate_factory, ffs_factory)
+        self.auth_plugin = auth_plugin
+
+        if self.auth_plugin is not None:
+            log(INFO, "Starting Exec API with user authentication")
```

Comment:
I'd prefer to move this log outside of the constructor. The constructor is not where the API gets started.

Reply:
Great catch! Make sense to me.